checks/raw/fuzzing_test.go

```diff
@@ -216,14 +216,14 @@ func Test_fuzzFileAndFuncMatchPattern(t *testing.T) {
 				t.Errorf("retrieve supported language error")
 			}
 			fileMatchPattern := langSpecs.fuzzFileMatchPattern
-			fileMatch, _ := path.Match(fileMatchPattern, tt.fileName)
-			if (fileMatch != tt.expectedFileMatch) && !tt.wantErr {
+			fileMatch, err := path.Match(fileMatchPattern, tt.fileName)
+			if fileMatch != tt.expectedFileMatch || err != nil && !tt.wantErr {
```

Comment:
maybe add some parenthesis around the if statement. It's unclear whether it means
```
fileMatch != tt.expectedFileMatch || (err != nil && !tt.wantErr)
```

or 
```
(fileMatch != tt.expectedFileMatch || err != nil) && !tt.wantErr
```